tendrl/commons/objects/__init__.py

```diff
@@ -75,94 +75,27 @@ def load_definition(self):
-                                    types.MethodType, types.BuiltinMethodType,
-                                    types.UnboundMethodType)) or \
-                        attr.startswith("_") or attr in ['value', 'list']:
-                    continue
-
-                if val is None and hasattr(current_obj, attr):
-                    # if self.attr is None, use attr value from central
-                    # store (i.e. current_obj.attr)
-                    if getattr(current_obj, attr):
-                        setattr(self, attr, getattr(current_obj, attr))
-
-        self.updated_at = str(time_utils.now())
-        for item in self.render():
-            '''
-                Note: Log messages in this file have try-except
-                blocks to run
-                in the condition when the node_agent has not been
-                started and
-                name spaces are being created.
-            '''
-            try:
-                Event(
-                    Message(
-                        priority="debug",
-                        publisher=NS.publisher_id,
-                        payload={"message": "Writing %s to %s" %
-                                            (item['key'], item['value'])
-                                 }
-                    )
-                )
-            except KeyError:
-                sys.stdout.write("Writing %s to %s" % (item['key'],
-                                                       item['value']))
-            # convert list, dict (json) to python based on definitions
-            _type = self._defs.get("attrs", {}).get(item['name'],
-                                                    {}).get("type")
-            if _type:
-                if _type.lower() in ['json', 'list']:
-                    if item['value']:
-                        try:
-                            item['value'] = json.dumps(item['value'])
-                        except ValueError as ex:
-                            _msg = "Error save() attr %s for object %s" % \
-                                   (item['name'], self.__name__)
-                            Event(
-                                ExceptionMessage(
-                                    priority="debug",
-                                    publisher=NS.publisher_id,
-                                    payload={"message": _msg,
-                                             "exception": ex
-                                             }
-                                )
-                            )
-            try:
-                NS._int.wclient.write(item['key'], item['value'], quorum=True)
-            except (etcd.EtcdConnectionFailed, etcd.EtcdException):
-                NS._int.wreconnect()
-                NS._int.wclient.write(item['key'], item['value'], quorum=True)
-        if ttl:
-            etcd_utils.refresh(self.value, ttl)
+        key = self.value + "/data"
```

Comment:
whats happening here? Seems like a major change to the way tendrl objects are stored in etcd